dfaker.py

```diff
@@ -12,18 +12,20 @@
 # -g,           --gaps               Add gaps to fake data
 # -s SMBG_FREQ, --smbg SMBG_FREQ     Freqency of fingersticks a day: high, average or low
 
-from datetime import datetime, timedelta, tzinfo
-import time
+from datetime import datetime
 from pytz import timezone
```

Comment:
I don't think you're using `timezone` directly, so no need to import it like this since you're importing all of `pytz` on the next line...